bindings/wallet-jni/java/run_test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import subprocess
+import sys
+
+scriptdirectory = Path(__file__).parent
+rootdirectory = scriptdirectory.parent.parent.parent
+dynlibdirectory = rootdirectory / "target" / "debug"
+
+junit = "junit-4.13.jar"
+hamcrest = "hamcrest-core-1.3.jar"
+
+classpath = f".:{junit}:{hamcrest}"
+javafilesdir = Path("com/iohk/jormungandrwallet")
+
+
+def compile():
+    packageFiles = map(lambda path: str(path), (scriptdirectory /
```

Comment:
nitpicking: `map(str, ...)`

Reply:
good point

Reply:
I'm still not sure if I want to keep this as a python script, though. Maybe something like gradle would work here, but I don't have much experience with it.

Reply:
Yeah, maybe gradle would be better from a java approach. I have not much experience myself too. I have seen in some posts that people use gradle to even call `cargo` so maybe at some point we can use it to pack it all together. I don't think we should lose too much time with that now so I guess the python script is enough. What do yo think @NicolasDP ?